### PR TITLE
worked on html

### PR DESCRIPTION
**Pull request by lilvina**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+<head>
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link rel="stylesheet" href="public/stylesheets/styles.css">
+  <title>My Calculator</title>
+</head>
+<body>
+  <div class="calculator">
+  </div>
+  <div class="row">
+    <div class="row1">
```
Comment (jrobcodes) [inline index.html]: You could greatly simplify this HTML by removing many of these divs, using more targeted CSS selectors, and placing all of these divs under the `calculator` div (lines 9 through 10). I don't think any of the internal divs would be necessary - you could just create the buttons inline and position them with CSS:
```html
<div class="calculator">
  <p class="calculator-button">AC</p>
  <!-- all the buttons here -->
</div>
```
Note that the requirements state that class names are namespaced under the word "calculator", so `row` and `button-font` would be invalid class names. Finally, I would consider a different tag for the individual buttons - `p` (paragraph) doesn't seem semantically valid for this context.

**Review by jrobcodes — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+<body>
+  <div class="background">
+  </div>
+
+  <div class="row1">
+    <div class="button-calculator">
+      <button class="AC">AC</button>
+    </div>
+    <div class="button-calculator">
+      <button class="positive/negative">+/-</button>
```
Comment (FrankieFabuloso) [inline index.html:17]: I would avoid putting '/' in any kind of identifier name just cause it has different meaning based on context. 
The convention seems to be that you write multi-word identifier names like:
class='I-am-more-than-one-word'

Comment (jrobcodes) [reply index.html:17]: The convention used by google is lower snake case: `i_am_more_than_one_word`, or in this case, `positive_negative`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <title>My Calculator</title>
+</head>
+<body>
+  <div class="background">
```
Comment (FrankieFabuloso) [inline index.html:9]: Usualy when you want to center something on a browser window you wrap all the elements in a `<div class='container'></div>`.
So I suggest changing `background` to `container` and wrapping all of your `rows` in `<div class='container'>`

File: index.html (hunk 2 above)
Comment (FrankieFabuloso) [reply index.html:17]: That way your margin: auto will work in your CSS. :)

**Pull request merged**